logger.go

```diff
@@ -266,7 +286,8 @@ func (l *Logger) Log(severity int, msg string, fields map[string]interface{}) er
 		l.mu.Lock()
 		defer l.mu.Unlock()
 		if _, err := l.output.Write(b); err != nil {
-			fmt.Fprintf(os.Stderr, "logger output causes an error: %v", err)
+			l.handleError(err)
+			fmt.Fprintf(os.Stderr, "logger output causes an error: %v\n", err)
```

Comment:
`fmt.Fprintf(...)` can be moved to the default error handler.
`errors.Wrap(...)` can also be returned from the default error handler.

if _, err := l.output.Write(b); err != nil {
    return l.handleError(err)
}
```